docs/cookbook/invoices/custom-invoice.rst

```diff
@@ -0,0 +1,197 @@
+How to customize the order invoice?
+===================================
+
+.. note::
+
+    This cookbook describes customization of a feature available only with `Sylius/InvoicingPlugin <https://github.com/Sylius/InvoicingPlugin/>`_ installed.
+
+The invoicing plugin lets you generate invoice on admin panel and download PDF with it. This plugin also sends an email with this invoice
+as well as let the administrator to delegate an email resend.
+
+Why would you customize the invoice?
+------------------------------------
+
+Invoicing Plugin provides a generic solution for generating invoices for orders, it is enough for a basic invoicing functionality
+but many shops need this feature customized for its needs.
+For example, one may need to change the look of it, or add some more data.
+
+Getting started
+---------------
+
+Before you start make sure that you have:
+1.`Sylius/InvoicingPlugin <https://github.com/Sylius/InvoicingPlugin/>`_ installed.
+2.`Wkhtmltopdf <https://wkhtmltopdf.org/>`_ package installed, because most of pdf generation is done with it.
+
+How to customize the invoice appearance?
+----------------------------------------
+
+There might be need to change how the invoices look like, f.e. shop is using its own logo, there might be some colours changed
+on the tables, or maybe the order of fields might be changed.
+
+First let's prepare the HTML's so we can modify them. This command will copy all of the templates:
+
+.. code-block:: bash
+
+    mkdir templates/bundles/SyliusInvoicingPlugin
+    cp -r vendor/sylius/invoicing-plugin/src/Resources/views/Invoice templates/bundles/SyliusInvoicingPlugin/
```

Comment:
great idea with this command, but you should copy only modified files (now when we change sth in Invoice Plugin, the user will overwrite it even if he don't customize anything)